voidcrew/modules/shuttle/engine/fuel.dm

```diff
@@ -10,42 +10,82 @@
 
 	icon_state_off = "burst_plasma_off"
 
-	///The specific gas to burn out of the engine heater. If none, burns any gas.
```

Comment:
again, I think we should just let it work (it would still be not constructible in-game) so we don't have to worry about problems if it's spawned in for unit testing and such. Would also make it an admin item that can be given to players if they want to burn o2 or something lmao